docker/build.sh

```diff
@@ -34,14 +34,29 @@ done
 shift $((OPTIND - 1))
 
 #########Set Me###############
-export PYVER=${PYVER:-3.5}
+export PYVER=${PYVER:-3.6}
 export PYV=${PYVER/./}
 export CUDA_VERSION=${CUDA_VERSION:-10}
+
+if [ "${CUDA_VERSION%%\.*}" ]
+then
+  if [ $CUDA_VERSION != "9" ] && [ $CUDA_VERSION != "10" ]
+  then
+      echo "Wrong CUDA_VERSION=$CUDA_VERSION provided. Only 9 and 10 are supported"
+      exit 1
+  fi
+else
+  export CUDA_VERSION=${CUDA_VERSION#*\.}
+  echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+  echo "Forcing $CUDA_VERSION. Make sure that Dockerfile.cuda$CUDA_VERSION.deps is provided"
+  echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+fi
+
```

Comment:
By any chance user can use already built docker image? After all, that's the purpose of docker...
E.g. if he does `docker pull nvidia/cuda:latest`, he doesn't need to have `Dockerfile.cuda10.deps` and build another artificial image.

Reply:
This currently assumes that the user needs to specify manually CUDA version, so even if he has `nvidia/cuda:latest` he needs to tag it as `nvidia/dali:cuda${CUDA_VERSION}.toolkit` for consistency. The main reason is that depending on a particular CUDA version we do some stuff differently (like building TF plugins).